### PR TITLE
Add an easing setting

### PR DESCRIPTION
**Pull request by SeinopSys**
I added this in a version that I use in one of my projects, thought I'd make a PR in case you think this option is worth adding. I really like using the `ease-in-out` timing function and `delayBetweenCycles` together for a nice speed up & slow down between cycles.

### CODE REVIEW

File: lib/jquery.simplemarquee.js [hunk 1]
```diff
@@ -254,7 +255,7 @@
 
         // Append the style and associate the animation to the wrappers
         this._element.append(styleStr);
-        this._wrappers.css('animation', this._animationName + ' ' + (this._size / this._options.speed) + 's linear infinite');
+        this._wrappers.css('animation', this._animationName + ' ' + (this._size / this._options.speed) + 's '+this._options.easing+' infinite');
```
Comment (satazor) [inline lib/jquery.simplemarquee.js]: Please add a space beetwen ' and +

**Comment by satazor** [comment]
Great addition. Can you also update the README?

**Comment by SeinopSys** [comment]
Added spaces & I believe I already updated the README.

**Comment by satazor** [comment]
@SeinopSys you are right!

**Comment by satazor** [comment]
Released as `0.3.0`, ty for you contribution.

**Pull request merged**